binder-requirements.txt

```diff
@@ -0,0 +1,4 @@
+qiskit-rigetti==0.4.*
```

Comment:
Maybe worth a note about updating this and checking the docs in the development section of the README?